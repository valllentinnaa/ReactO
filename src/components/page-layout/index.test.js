import React from 'react'
import renderer from 'react-test-renderer'
import PageLayout from '.'
import TestingEnvironment from '../../test-utils/router'
jest.mock('../header', () => 'Header');
jest.mock('../side-navigation/SideNavigation', () => 'SideNavigation');
jest.mock('../Footer', () => 'Footer');

describe('PageLayout Component', () => {
  it('should render pagelayout component', () => {
  
    
    const tree = renderer.create(
      <TestingEnvironment value={{
        user: {
          loggedIn: true,
          id: '123'
        }
      }}>
        <PageLayout />
      </TestingEnvironment>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})