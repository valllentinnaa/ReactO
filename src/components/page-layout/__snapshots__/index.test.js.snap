// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLayout Component should render pagelayout component 1`] = `
Array [
  <Header />,
  <div
    className="body-container"
  >
    <SideNavigation />
  </div>,
  <Footer />,
]
`;
